src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -383,6 +388,10 @@ private void UpdatePatchDetails()
             percentageFormat.FormatSafe(Math.Round(GetCompoundAmount(SelectedPatch, carbondioxideCompound),
                 Constants.ATMOSPHERIC_COMPOUND_DISPLAY_DECIMALS));
 
+        otherCompoundLabel.Text = percentageFormat.FormatSafe(100.0f - Math.Round(GetCompoundAmount(SelectedPatch, carbondioxideCompound)
+            + GetCompoundAmount(SelectedPatch, nitrogenCompound) + GetCompoundAmount(SelectedPatch, oxygenCompound)),
+            Constants.ATMOSPHERIC_COMPOUND_DISPLAY_DECIMALS);
```

Comment:
I'm not fully sure myself, but can the rounding mode end up in a situation where the rounded numbers add up to slightly more than 100%? Like 100.1%. I wonder if there is a way to round down this to the display precision to ensure this never overshoots.

Reply:
No, like a floor operation but just at the display decimal count - 1 and not towards the lower integer like `Floor` by default does. Here's a discussion related to this, but it is for python: https://stackoverflow.com/questions/58065055/floor-and-ceil-with-number-of-decimals though porting the code over to C# doesn't seem impossible.

Reply:
Also the other label should not show up at all if there's nothing in there. It makes no sense to say "btw these above numbers add up to 100% so there's no remainder left".